pages/c/_cluster/_product/projectsnamespaces.vue

```diff
@@ -79,7 +79,8 @@ export default {
     },
     projectsWithoutNamespaces() {
       return this.clusterProjects
-        .filter(project => !this.projectIdsWithNamespaces.includes(project.name));
+        .filter(project => this.projectIdsWithNamespaces.findIndex(item => item.includes(project.id)) > -1

```

Comment:
This could be simplified/improved (findIndex --> find, includes --> endsWith)
```
    projectsWithoutNamespaces() {
      return this.clusterProjects.filter(project => !!this.projectIdsWithNamespaces.find(item => item.endsWith(`/${ project.id }`)));
    },
```